lib/btrfs_test.pm

```diff
@@ -18,13 +19,53 @@ sub set_unpartitioned_disk_in_bash {
         $vd = 'sd';
     }
     assert_script_run 'parted --script --machine -l';
-    assert_script_run 'disk=${disk:-$(parted --script --machine -l |& sed -n \'s@^\(/dev/' . $vd . '[ab]\):.*unknown.*$@\1@p\')}';
+    assert_script_run 'disk=${disk:-$(parted --script --machine -l |& sed -n \'s@^\(/dev/' . $vd . 'b\):.*$@\1@p\')}';
```

Comment:
That was a bad idea -> https://openqa.suse.de/tests/1007823#step/btrfs_qgroups/12 Didn't you read the git log of that file in before? We have a weirdly behaving qemu that is making the second disk /dev/vda and the other way around. I will revert that commit for now.